src/main/java/CashDrawer.java

```diff
@@ -0,0 +1,82 @@
+public class CashDrawer {
+    // Include Cash values to build a CashDrawer Array of a Cash Register
+    int twenty;
```

Comment:
Should these have private of public or package access modifiers? Rec you make them private.